src/emmy/differential.cljc

```diff
@@ -13,6 +13,7 @@
   (:refer-clojure :exclude [compare])
   (:require [clojure.core :as core]
             [clojure.string :refer [join]]
+            [emmy.function]  ;; for the side effect of making kind: MultiFn -> ::v/function
```

Comment:
::g/function?

Reply:
I think `::v/function` is the correct spelling here